Master-Game/Entities/Enemy.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+namespace MasterGame.Entities
+{
+    public class Enemy : HumanEntity
+    {
+        public Enemy()
+            : base ("Evil Guy", 500)
```

Comment:
Will need to allow people to indicate their name and health. As it stands, every time an enemy is created, the name and health will always be "Evil Guy" and 500, respectively.